app-frontend/src/app/components/featureFlagOverrides/featureFlagOverrides.html

```diff
@@ -0,0 +1,28 @@
+<div class="feature-flags">
+  <h5>Feature Flags</h5>
+  <div id="feature-flag--{{flag.key}}" class="feature-flags-flag" ng-repeat="flag in $ctrl.flags">
+    <div class="feature-flags-name">Feature Name: {{flag.name || flag.key}}</div>
+    <div class="feature-flags-desc">Description: {{flag.description}}</div>
+    <div class="feature-flags-actions">
+      Flag:
+      <button id="feature-flag--{{flag.key}}--enable"
```

Comment:
hm, it would shorten the view, but it doesn't really fit with the way I have the data formatted - active state is applied using an ng-class which has a conditional, where the uib-btn-radio expects an ng-model. Using the radio buttons would mean that I have to manage state for both the buttons and the feature flags inside the controller, whereas right now, it's just calling service methods to determine what state to display and to set state.